docs/create-packages/Explicit-assembly-reference.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Explicit Assembly References
```

Comment:
If it's not too difficult, I'd suggest trying to clarify the customer task in the title metadata and the H1 (possibly in the TOC, too, if it's not too long). It looks like the customer task might be something like "Update reference assemblies for use with PackageReference". I'm not sure if that's quite right.

Reply:
What do you think about "Prevent some assemblies being referenced by projects"? Maybe "package assemblies" instead of just "assemblies". But I think "prevent" sounds too negative, I don't really like it.

I'm really struggling to think of a short title, particularly for the ToC. The page is about how package authors need to create their package so that only a subset of the assemblies in the package are "referenced" by projects that use the package. That's far too long for a title. The page explains for all project types, so we shouldn't need to call out either PackageReference or packages.config in the title, they're different sections on the page.

Reply:
I agree that title is too negative, so if there's not a good way to phrase the customer task, I wouldn't change anything from the current title/H1.  (I wouldn't worry about the TOC though. It doesn't necessarily need to match the H1 or title field exactly.) Possibly, something fairly generic would be OK, too. Like "Configure|Modify|Update package assembly references". If that is not good either, I would leave it as-is.

Reply:
I need to write down this idea before I forget it. Something like "Selecting assemblies referenced by projects". I hope that fits nicely in the ToC :)  Maybe I'll have time to update the PR this afternoon, otherwise on Monday.